manifests/plugin/deb.pp

```diff
@@ -1,5 +1,33 @@
 # @summary Pulp Deb plugin
+# @param use_pulp2_content_route
```

Comment:
Could you add a short description? It'll end up being rendered something like https://forge.puppet.com/theforeman/pulpcore/reference#pulpcorepluginrpm